### PR TITLE
[ODS-4237]  Ed-Fi-Alliance repo paths in nuspec files

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: Application/EdFi.Admin.DataAccess/EdFi.Admin.DataAccess.csproj [hunk 1]
```diff
@@ -7,6 +7,7 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <Copyright>Copyright © 2020 Ed-Fi Alliance, LLC and Contributors</Copyright>
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+	<PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
```
Comment (jamessmckay) [inline Application/EdFi.Admin.DataAccess/EdFi.Admin.DataAccess.csproj:10]: fix formatting

File: Application/EdFi.Ods.Common/EdFi.Ods.Common.csproj [hunk 2]
```diff
@@ -36,4 +37,7 @@
   <ItemGroup>
     <ProjectReference Include="..\EdFi.Common\EdFi.Common.csproj" />
   </ItemGroup>
+  <ItemGroup>
```
Comment (jamessmckay) [inline Application/EdFi.Ods.Common/EdFi.Ods.Common.csproj:40]: fix formatting

File: Utilities/CodeGeneration/EdFi.Ods.CodeGen/EdFi.Ods.CodeGen.csproj [hunk 3]
```diff
@@ -67,4 +67,7 @@
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+  <ItemGroup>
```
Comment (jamessmckay) [inline Utilities/CodeGeneration/EdFi.Ods.CodeGen/EdFi.Ods.CodeGen.csproj:70]: fix formatting

File: Utilities/DataLoading/EdFi.LoadTools/EdFi.LoadTools.csproj [hunk 4]
```diff
@@ -9,6 +9,8 @@
         <RootNamespace>EdFi.LoadTools</RootNamespace>
         <Copyright>Copyright © 2020 Ed-Fi Alliance, LLC and Contributors</Copyright>
         <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+		<PackageProjectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/tree/master/Utilities/DataLoading</PackageProjectUrl>
```
Comment (jamessmckay) [inline Utilities/DataLoading/EdFi.LoadTools/EdFi.LoadTools.csproj:12]: fix formatting

File: Utilities/DataLoading/EdFi.LoadTools/EdFi.LoadTools.csproj [hunk 5]
```diff
@@ -29,4 +31,7 @@
     <ItemGroup>
       <Folder Include="Properties\" />
     </ItemGroup>
+	<ItemGroup>
```
Comment (jamessmckay) [inline Utilities/DataLoading/EdFi.LoadTools/EdFi.LoadTools.csproj:34]: fix formatting

File: Utilities/DataLoading/EdFi.SmokeTest.Console/EdFi.SmokeTest.Console.nuspec [hunk 6]
```diff
@@ -5,6 +5,7 @@
     <version>0.0.0</version>
     <authors>Ed-Fi Alliance</authors>
     <owners>Ed-Fi Alliance</owners>
+	<projectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS</projectUrl>
```
Comment (jamessmckay) [inline Utilities/DataLoading/EdFi.SmokeTest.Console/EdFi.SmokeTest.Console.nuspec:8]: fix formatting

File: Utilities/SdkGen/EdFi.SdkGen.Console/EdFi.OdsApi.Sdk.nuspec [hunk 7]
```diff
@@ -6,6 +6,7 @@
     <title>EdFi.OdsApi.Sdk</title>
     <authors>$authors$</authors>
     <owners>$owners$</owners>
+	<projectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS</projectUrl>
```
Comment (jamessmckay) [inline Utilities/SdkGen/EdFi.SdkGen.Console/EdFi.OdsApi.Sdk.nuspec:9]: fix formatting

File: Application/EdFi.Admin.DataAccess/EdFi.Admin.DataAccess.csproj (hunk 1 above)
Comment (semalaiappan) [reply Application/EdFi.Admin.DataAccess/EdFi.Admin.DataAccess.csproj]: Fixed it . Thanks

File: Application/EdFi.Ods.Common/EdFi.Ods.Common.csproj (hunk 2 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.Common/EdFi.Ods.Common.csproj:40]: Fixed it . Thanks

File: Utilities/DataLoading/EdFi.LoadTools/EdFi.LoadTools.csproj (hunk 4 above)
Comment (semalaiappan) [reply Utilities/DataLoading/EdFi.LoadTools/EdFi.LoadTools.csproj]: Fixed it . Thanks

File: Utilities/DataLoading/EdFi.SmokeTest.Console/EdFi.SmokeTest.Console.nuspec (hunk 6 above)
Comment (semalaiappan) [reply Utilities/DataLoading/EdFi.SmokeTest.Console/EdFi.SmokeTest.Console.nuspec]: Fixed it . Thanks

File: Utilities/SdkGen/EdFi.SdkGen.Console/EdFi.OdsApi.Sdk.nuspec (hunk 7 above)
Comment (semalaiappan) [reply Utilities/SdkGen/EdFi.SdkGen.Console/EdFi.OdsApi.Sdk.nuspec]: Fixed it . Thanks

File: Application/EdFi.Ods.Common/EdFi.Ods.Common.csproj (hunk 2 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.Common/EdFi.Ods.Common.csproj:40]: Fixed it . Thanks

File: Utilities/CodeGeneration/EdFi.Ods.CodeGen/EdFi.Ods.CodeGen.csproj (hunk 3 above)
Comment (semalaiappan) [reply Utilities/CodeGeneration/EdFi.Ods.CodeGen/EdFi.Ods.CodeGen.csproj:70]: Fixed it . Thanks

File: Application/EdFi.Ods.Common/EdFi.Ods.Common.csproj (hunk 2 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.Common/EdFi.Ods.Common.csproj:40]: Fixed it . Thanks

File: Utilities/DataLoading/EdFi.LoadTools/EdFi.LoadTools.csproj (hunk 4 above)
Comment (semalaiappan) [reply Utilities/DataLoading/EdFi.LoadTools/EdFi.LoadTools.csproj]: Fixed it . Thanks

File: Utilities/DataLoading/EdFi.LoadTools/EdFi.LoadTools.csproj (hunk 5 above)
Comment (semalaiappan) [reply Utilities/DataLoading/EdFi.LoadTools/EdFi.LoadTools.csproj]: Fixed it . Thanks

File: Utilities/DataLoading/EdFi.SmokeTest.Console/EdFi.SmokeTest.Console.nuspec (hunk 6 above)
Comment (semalaiappan) [reply Utilities/DataLoading/EdFi.SmokeTest.Console/EdFi.SmokeTest.Console.nuspec]: Fixed it . Thanks

File: Utilities/SdkGen/EdFi.SdkGen.Console/EdFi.OdsApi.Sdk.nuspec (hunk 7 above)
Comment (semalaiappan) [reply Utilities/SdkGen/EdFi.SdkGen.Console/EdFi.OdsApi.Sdk.nuspec]: Fixed it . Thanks

**Comment by semalaiappan** [comment]
All formatting issue are fixed . I used 2 spaces for tabs in xml only

File: Application/EdFi.Admin.DataAccess/EdFi.Admin.DataAccess.csproj [hunk 8]
```diff
@@ -21,5 +22,6 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\EdFi.Common\EdFi.Common.csproj" />
+    <None Include="../../LICENSE.txt" Pack="true" PackagePath="LICENSE.txt" />
```
Comment (jamessmckay) [inline Application/EdFi.Admin.DataAccess/EdFi.Admin.DataAccess.csproj:25]: this should be in another item group. lets keep project references together in their own group

File: Application/EdFi.Ods.Common/EdFi.Ods.Common.csproj [hunk 9]
```diff
@@ -35,5 +36,6 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\EdFi.Common\EdFi.Common.csproj" />
+    <None Include="../../LICENSE.txt" Pack="true" PackagePath="LICENSE.txt" />
```
Comment (jamessmckay) [inline Application/EdFi.Ods.Common/EdFi.Ods.Common.csproj:39]: same thing as above

File: Utilities/CodeGeneration/EdFi.Ods.CodeGen/EdFi.Ods.CodeGen.csproj [hunk 10]
```diff
@@ -44,21 +41,18 @@
     <PackageReference Include="System.Configuration.ConfigurationManager" Version="4.7.0" />
     <PackageReference Include="System.Data.SqlClient" Version="4.8.1" />
   </ItemGroup>
-
   <ItemGroup>
     <ProjectReference Include="..\..\..\Application\EdFi.Ods.Common\EdFi.Ods.Common.csproj" />
+    <None Include="../../../LICENSE.txt" Pack="true" PackagePath="LICENSE.txt" />
```
Comment (jamessmckay) [inline Utilities/CodeGeneration/EdFi.Ods.CodeGen/EdFi.Ods.CodeGen.csproj:46]: this should move the group with the other nones or contents

File: Application/EdFi.Admin.DataAccess/EdFi.Admin.DataAccess.csproj (hunk 8 above)
Comment (semalaiappan) [reply Application/EdFi.Admin.DataAccess/EdFi.Admin.DataAccess.csproj:27]: Fixed it . Thanks

File: Application/EdFi.Ods.Common/EdFi.Ods.Common.csproj (hunk 9 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.Common/EdFi.Ods.Common.csproj:41]: Fixed it . Thanks

File: Utilities/CodeGeneration/EdFi.Ods.CodeGen/EdFi.Ods.CodeGen.csproj (hunk 10 above)
Comment (semalaiappan) [reply Utilities/CodeGeneration/EdFi.Ods.CodeGen/EdFi.Ods.CodeGen.csproj]: Fixed it . Thanks

**Pull request merged**